src/fondant/pipeline.py

```diff
@@ -18,8 +17,6 @@
 from fondant.schema import validate_partition_number, validate_partition_size
 
 if is_kfp_available():
```

Comment:
Ideally we can get rid of this and make this file completely runner independent.

Reply:
This is only needed for setting `p_volumes` at the `componentOp` level, that was only needed when we were initially planning to download the Laion dataset locally to use it. I think we can remove both `ephemeral_storage_size` and `p_volumes` from the ComponenOp. We can always re-add them later if we notice they're needed